### DISCUSSION TITLE
Obsolete ani-cli search function gives no results for anime. What happened?

### DISCUSSION THREAD

**Discussion by blastedbiggs** [pystardust/ani-cli#946] [Q&A] [answered]
I have a old device running Gentoo which has not been updated for roughly a year. Now, it appears that the ani-cli search function gives no results (yes, the device is connected to the internet). I'm out of the loop with this project so I'm curious what changes would have happened in a year for the older script to become inoperable. Thanks in advance.

**Comment by blastedbiggs** [comment]
Also, I understand that to solve this issue I simply update ani-cli. I'm mostly curious for the sake of learning more about scripting and this project in general. Thanks again.

**Comment by port19x** [comment, marked as answer]
This likely happened because thats generally how webscraping works.
Webscraping means extracting data from places on the web that don't want it to be extracted.
Depending on what service you're scraping, they will resist by changing things up a lot.
It's a cat and mouse game where scrapers, especially those with only one source, need to adapt frequently.
@blastedbiggs

**Reply by blastedbiggs** [reply]
Noted, thank you for your response.
